### PR TITLE
Review Criteria

### PR DESCRIPTION
**Pull request by Kechr**
Objectives, Initial Requirements Analysis, and Use Case Diagram

### CODE REVIEW

File: project_proposal/tex/mainpage.tex [hunk 1]
```diff
@@ -155,10 +166,13 @@ \subsection{Use Case Diagram}
 \end{figure}
 
 \subsection{Supplementary Requirements}
+FURPS is a model for classifying functional and non-functional requirements. The
+acronym stand for:
```
Comment (kkmp-dk) [inline project_proposal/tex/mainpage.tex]: Is the acronym missing?

Comment (Kechr) [reply project_proposal/tex/mainpage.tex]: Not quite sure what I was thinking here. It's been rewritten, does it make sense now?

Comment (kkmp-dk) [reply project_proposal/tex/mainpage.tex]: It does make senese

File: project_proposal/tex/mainpage.tex [hunk 2]
```diff
@@ -108,45 +108,56 @@ \subsection{Summary of requirements}
 The system must be able to find the optimal production speed for each product type, based on an error simulation and the appertaining graph upon which the error simulation is built.
 
 \subsection{List of requirements}
+Below is a list of the above requirements. These requirements have been
+prioritised using the MoSCoW method, where M is for Must have, S is for
+Should have, C is for could have, and W is for Won't have. 
+
 \begin{table}[H]
-    \begin{tabularx}{\textwidth}{|>{\RaggedRight}p{1cm}|>{\RaggedRight}p{4cm}|>{\RaggedRight}X|}
+    \begin{tabularx}{\textwidth}{|>{\RaggedRight}p{1cm}|>{\RaggedRight}p{4cm}|>{\RaggedRight}X|>{\RaggedRight}p{1cm}}
```
Comment (kkmp-dk) [inline project_proposal/tex/mainpage.tex]: You are missing this | in the end 
```
\begin{tabularx}{\textwidth}{|>{\RaggedRight}p{1cm}|>{\RaggedRight}p{4cm}|>{\RaggedRight}X|>{\RaggedRight}p{1cm}|}
```

File: project_proposal/tex/mainpage.tex [hunk 3]
```diff
@@ -108,45 +108,56 @@ \subsection{Summary of requirements}
...
+        \hline
+        \textbf{ID} & \textbf{Name} & \textbf{Description} & M \\
```
Comment (kkmp-dk) [inline project_proposal/tex/mainpage.tex]: should this line be like this? 

or like this? 
image

**Review by kkmp-dk — CHANGES_REQUESTED** [review]

File: project_proposal/tex/mainpage.tex (hunk 2 above)
Comment (Kechr) [reply project_proposal/tex/mainpage.tex]: Good catch.

File: project_proposal/tex/mainpage.tex (hunk 3 above)
Comment (kkmp-dk) [reply project_proposal/tex/mainpage.tex]: hey this still says M instead of Priority 
Is it supposed to say that?

**Review by kkmp-dk — APPROVED** [review]

**Comment by Kechr** [comment]
If ready to merge, please do so. @kepet19

**Pull request merged**